services/tenant-ui/frontend/src/components/connections/Connections.vue

```diff
@@ -45,13 +45,17 @@
             :connection-name="data.alias"
           />
           <Button
+            v-if="config.frontend.showWritableComponents"
             title="Delete Contact"
             icon="pi pi-trash"
             class="p-button-rounded p-button-icon-only p-button-text"
             :disabled="deleteDisabled(data.alias)"
             @click="deleteContact($event, data.connection_id)"
           />
-          <EditContact :connection-id="data.connection_id" />
+          <EditContact
+            v-if="config.frontend.showWritableComponents"
+            :connection-id="data.connection_id"
+          />
```

Comment:
Not sure we want to remove connection management functionality? Not specified in ticket so maybe a quick question for Emiliano.

Reply:
I think connection management should stay, as connections are often handled out-of-band (i.e.: by hand) before two agents can communicate.

Reply:
Done.